templates/application/_base.scss

```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ * Application template
+ *
+*/
+
+.info-banner {
+	background-color: $alert-info-bg;
+	color: $text-color;
+	font-size: 20px;
+	line-height: 1.65em;
+	padding: 15px 0;
+
+	h2 {
+		float: left;
+		font-size: 1em;
+		line-height: 1.65em;
+		margin: 0 .25em 0 0;
+
+		&:after {
+			content: ":";
+			margin-left: .125em;
+		}
+	}
+
+	.info-banner-actions {
+		align-items: center;
+		display: flex;
+		justify-content: space-between;
+	}
+}
+
+.application-bar {
+	background-color: #38414d;
+	color: #fff;
+	margin-top: 15px;
+
+	h2 {
```

Comment:
Should the font size be fixed at 28px and 18px depending of the view port and 34px for print media?

* https://github.com/wet-boew/cdts-sgdc/blob/master/public/gcweb/cdtsapps.css#L309
* https://github.com/wet-boew/cdts-sgdc/blob/master/public/gcweb/cdtsapps.css#L123
* https://github.com/wet-boew/cdts-sgdc/blob/master/public/gcweb/cdtsapps.css#L363